website/content/about/working-groups/governance.md

```diff
@@ -4,17 +4,114 @@ linkTitle: Governance
 url: /about/governance/
 ---
 
+{{% blocks/lead color="primary" align="left" %}}
 
-We aim to assist all CNCF projects with implementing good open source governance
-practices.
+Want to implement good open source governance? We can assist you with best practices, resources, and guidance on how!
 
-We meet once a month.
-Go to <a href="https://www.cncf.io/calendar/">cncf.io/calendar</a> and enter "Governance WG" in the search box to see when we meet next.
-    
-* [Zoom](https://zoom.us/my/cncftagcontributorstrategy?pwd=TnI0WU9Eb2I1RlRWdkl1R0k1WkZXUT09) passcode: `77777`
-* [Meeting Minutes and Agenda](https://docs.google.com/document/d/1P9tQgCM6OwDHd1F8UnWuauL4KDVTMTp49_n64_w8nrs/edit)
+Read our [Charter](https://github.com/cncf/tag-contributor-strategy/blob/main/governance/README.md)
 
-Discussion happens on the [mailing list] or on #tag-contributor-strategy on [Slack].
+{{% /blocks/lead %}}
 
-[mailing list]: https://lists.cncf.io/g/cncf-tag-contributor-strategy
-[Slack]: https://slack.cncf.io/
+<div class="section-group">
+
+{{% blocks/section color="dark" %}}
+
+The Governance Growth Working Group aim to assist all CNCF projects with implementing good open source governance practices.
```

Comment:
```suggestion
The Governance Working Group assists CNCF projects with implementing good open source governance practices.
```

Reply:
Oops, now you know I copy pasted stuff from the other pages I prepared 🤣